src/lms-harmonizer/edfi_lms_harmonizer/helpers/argparser.py

```diff
@@ -0,0 +1,191 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Please add unit tests to cover this module.

Reply:
✔️